lib/views/details_page/details_page.dart

```diff
@@ -94,23 +94,26 @@ class _DetailsPageState extends State<DetailsPage> {
                             children: [
                               AddToListButton(
                                 game: game,
-                                cubit: context.watch<FavGamesCubit>(),
+                                cubit: context.watch<UserCubit>(),
+                                listName: UserListNames.favGames,
                                 removeTooltip: 'remove_fav'.tr,
                                 addTooltip: 'add__fav'.tr,
                                 icon: Icons.favorite_rounded,
                                 activeColor: Colors.red,
                               ),
                               AddToListButton(
                                 game: game,
-                                cubit: context.watch<PlayedGamesCubit>(),
+                                cubit: context.watch<UserCubit>(),
+                                listName: UserListNames.playedGames,
                                 removeTooltip: 'remove_played'.tr,
                                 addTooltip: 'add__played'.tr,
                                 icon: Icons.check,
                                 activeColor: Colors.green,
                               ),
                               AddToListButton(
                                 game: game,
-                                cubit: context.watch<WishlistGamesCubit>(),
+                                cubit: context.watch<UserCubit>(),
```

Comment:
I think you don't have to add this cubit three times to this widget. Just define that in AddToListButton widget.